core/src/usb.py

```diff
@@ -62,10 +86,9 @@
     usb21_landing=False,
 )
 bus.add(iface_wire)
-if __debug__:
+if __debug__ and ENABLE_IFACE_DEBUG:
     bus.add(iface_debug)
-if not utils.BITCOIN_ONLY:
+if ENABLE_IFACE_WEBAUTHN:
```

Comment:
Same as before, `if not utils.BITCOIN_ONLY and ENABLE_IFACE_WEBAUTHN` makes the compiler to throw it out.